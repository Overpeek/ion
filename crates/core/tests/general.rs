#[test]
fn simple_general() {}
