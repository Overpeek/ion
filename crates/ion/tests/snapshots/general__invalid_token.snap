---
source: crates/core/tests/general.rs
expression: "err.pretty_print(false, code, \"<code>\")"
---
error: Invalid token
 --> <code>:1:5
  |
1 | x = \
  |     ^

